# Hummingbird
NC_061156.1	55455600
NC_061157.1	77975161
NC_061158.1	62712309
NC_061159.1	65793045
NC_061160.1	60726905
NC_061161.1	59913004
NC_061162.1	55045248
NC_061163.1	52021744
NC_061164.1	53447792
NC_061165.1	50363522
NC_061166.1	51488711
NC_061167.1	49247934
NC_061168.1	42497472
NC_061169.1	39157883
NC_061170.1	42201620
NW_025896150.1	64034
NW_025896151.1	24112
NW_025896152.1	48026
NW_025896153.1	65458
NW_025896154.1	48274
NW_025896155.1	48582
NW_025896156.1	48166
NW_025896157.1	48518
NW_025896158.1	43483
NW_025896159.1	51168
NW_025896160.1	44439
NW_025896161.1	40676
NW_025896162.1	42676
NW_025896163.1	39579
NW_025896164.1	38055
NW_025896165.1	33047
NW_025896166.1	30347
NW_025896167.1	26083
NW_025896168.1	26059
NW_025896169.1	28676
NW_025896170.1	22402
NW_025896171.1	23179
NW_025896172.1	25409
NW_025896173.1	22868
NW_025896174.1	26537
NW_025896175.1	23991
NW_025896176.1	20001
NW_025896177.1	32638
NW_025896178.1	30343
NW_025896179.1	24911
NW_025896180.1	23676
NW_025896181.1	33110
NW_025896182.1	30099
NW_025896183.1	22450
NW_025896184.1	26634
NW_025896185.1	20770
NW_025896186.1	24214
NW_025896187.1	20404
NW_025896188.1	45700
NC_011220.1	8194
NC_011221.1	7686


submit to mole
expression ones 
G004648-maybe
G025463-bc nematocysts
G014551-? neurons?
G020424-No, not high enough expression
G025592
G013153-no, not high 
G021759

Find wnt3 on NCBI then blast to AEP

test of selection on amino acid sequences (Look into it!)

4-14-2025: Keep the yes or no for present lysine site on the sequnecne page

collapse the early rooted ones-- (non-opsin?) ASO-1 and ASO-II also collapse but annotate! tetraopsin (retinochrome, neuropsin), R-opsin, c-opsin, xenopsin, anthazoan cnidops(?), 
there is only one removed opsin! The other is still in cnidops
4-9-2025: 
NOT OPSINS
(tyramine receptor in corals?)
HV105.MN822248
HVAEP12.G022632
HOLI01874.G49006

(xenopsin?) 
HV105.MN822249
HVAEP1.G000950
HOLI01595.G13504
4-1-2025: made cat file, need to run SLURM
IQ tree, need to add sequences to fasta file
made: C:\Users\Rachel Parsons\Documents\25Opsin\RP_3Hydra_formatted_sequences.fasta 
using VS code Python script then manually removed blank lines
need to cat that with Kyles file but docs in Macias Munoz group on hummingbird dont have write permissions
Onces in one doc, need to run script for IQ tree

#Do i want to test for selection for amino acids--are there spots being conserved or
ask Maya for pass for Itol


#3-4-25: In IQ tree need to delete H vulgaris sequences
grep "hydra"
keep hydractinia but get rid of the hydra opsins

in excel tab delineated extracts the sequences from AEP and olig
put into word 
make into a fasta
cat the filtered file from kyle with the AEP

name something final_opsins_list.fasta

#3-3-25: 
PREDICTED: Hydra vulgaris opsin-3 (LOC105847721), mRNA

HOLI01635.G27067-> kept from Kevins check

HOLI00003.G5348-> kept from Kevins check

HOLI02256.G12488-> from aide's check

HOLI09158.G57575-> new from aide's check

HOLI01711.G55576-> new from aide's check

HOLI00331.G46284-> new from aide's check

#2-27-25: reverse complement for mega alignment along with one and two spoaces at a time to fgind best on ewith no stop asterisk
#2-11-2025: 

#2-10-2025: Checked the genome browser for AEP and see that some of the sequences are not annotated as "long" as they blast to. Means I have to pull them out of the Olig Database from the 
portal. Continued with the process of clearing out the duplicated or missing opsins in the list generated by Blast. Because we are looking at comparisons from originally 105, the movement to AEP means there could be some duplications, retrotranspositions, or false positives (?).

#I wonder what the difference is
